example-config/readme.md

```diff
@@ -0,0 +1,21 @@
+# Running `etesync` under `nginx` and `uwsgi`
+
+This configuration assumes that etesync server has been installed in the home folder of a non privileged user 
+called `EtesyncUser` following the instructions in <https://github.com/etesync/server-skeleton>. Also that static 
+files have been collected by running the following command from within the `etesync` folder.
```

Comment:
Maybe also mention it assumes your web server is running as group www-data (it's not the case in all distros, for me, for example, it's http).